packages/flutter_tools/lib/src/base/terminal.dart

```diff
@@ -288,11 +288,11 @@ class AnsiTerminal implements Terminal {
         // prompt ends with ': '
         logger.printStatus(': ', emphasis: true, newline: false);
       }
-      choice = await keystrokes.first;
+      choice = (await keystrokes.first).trim();
```

Comment:
Does this mean that we can't use e.g. spacebar as an input?

Reply:
yes - though this matches the restrictions that the resident runner has. This could be changed to strip off a trailing newline line if length > 1 instead? I'm not sure that would buy us that much

Reply:
Maybe worth adding this as a comment, though I'm not sure this would be the best spot?

The spacebar is pretty big on keyboards, so it might be convenient to use for something someday.